### PR TITLE
Create 2017-05-24-the-outline-storytelling-platform-built-with-elixir…

### PR DESCRIPTION
**Pull request by jesskry**
…-phoenix.md

### CODE REVIEW

File: source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+>—[ theoutline.com](https://theoutline.com/)
+
+In many ways, *The Outline* represents the future of digital media through and through. Beyond an editorial focus on where society is headed, the application’s technology embodies the stack of the future: highly available, scalable, and responsive – *The Outline* is powered by [Phoenix](http://www.phoenixframework.org/), a framework written in Elixir. 
+
+[DockYard](https://dockyard.com/) engineering was lucky enough to partner with *The Outline* team on this ambitious project last October. We didn’t develop any of the eye candy, but under the hood and behind the scenes, we helped implement best practice authentication, authorization, content editing, and performance benchmarking amongst other things.
+ 
+With CEO [Joshua Topolsky](https://twitter.com/joshuatopolsky) (co-founder of The Verge, Vox Media, and former Editor-in-Chief of Engadget) and CTO [Ivar Vong](https://twitter.com/ivarvong), alongside [the amazing team assembled](https://medium.com/@joshuatopolsky/getting-into-the-outline-7735a7743bc8#.lntg3krbl), we’re excited to see how *The Outline* pushes the bounds of digital journalism with Phoenix.
```
Comment (michaeldupuisjr) [inline source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md]: alongside [their amazing team](https://medium.com/@joshuatopolsky/getting-into-the-outline-7735a7743bc8#.lntg3krbl), we’re

File: source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Elixir/Phoenix is the perfect technology stack for companies looking to support:
```
Comment (michaeldupuisjr) [inline source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md]: Phoenix is the perfect backend framework for

File: source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+* Modern web apps built in JavaScript (Ember.js, React.js, Angular.js)
```
Comment (michaeldupuisjr) [inline source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md]: Modern, single page applications written in JavaScript (i.e., Ember.js, React.js, Angular.js)

File: source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* Real-time updates for web apps (e.g., chat, data updates without page refresh)
+* Fault tolerance that minimizes bottlenecks by isolating application errors and exceptions
+* The ability to support millions of concurrent connections
```
Comment (michaeldupuisjr) [inline source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md]: * [Millions of concurrent connections](http://www.phoenixframework.org/blog/the-road-to-2-million-websocket-connections)

File: source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+[DockYard](https://dockyard.com/) engineering was lucky enough to partner with *The Outline* team on this ambitious project last October. We didn’t develop any of the eye candy, but under the hood and behind the scenes, we helped implement best practice authentication, authorization, content editing, and performance benchmarking amongst other things.
+ 
+With CEO [Joshua Topolsky](https://twitter.com/joshuatopolsky) (co-founder of The Verge, Vox Media, and former Editor-in-Chief of Engadget) and CTO [Ivar Vong](https://twitter.com/ivarvong), alongside [the amazing team assembled](https://medium.com/@joshuatopolsky/getting-into-the-outline-7735a7743bc8#.lntg3krbl), we’re excited to see how *The Outline* pushes the bounds of digital journalism with Phoenix.
+
+Elixir/Phoenix is the perfect technology stack for companies looking to support:
+
+* Modern web apps built in JavaScript (Ember.js, React.js, Angular.js)
+* Real-time updates for web apps (e.g., chat, data updates without page refresh)
```
Comment (michaeldupuisjr) [inline source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md]: * Realtime messaging using [Phoenix channels](https://dockyard.com/blog/2016/08/09/phoenix-channels-vs-rails-action-cable) which updates data without page reloads

File: source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+* Fault tolerance that minimizes bottlenecks by isolating application errors and exceptions
```
Comment (michaeldupuisjr) [inline source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md]: * High availability and fault tolerance thanks to [Phoenix Presence](https://dockyard.com/blog/2016/03/25/what-makes-phoenix-presence-special-sneak-peek)

File: source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md (hunk 4 above)
Comment (jesskry) [inline source/posts/2017-05-24-the-outline-storytelling-platform-built-with-elixir-phoenix.md]: @michaeldupuisjr K. Changes merged if you wanted to take a final look before I ask for it to be published. Thanks!

**Comment by michaeldupuisjr** [comment]
@jesskry This looks good to me.

**Pull request closed without merging**